content/en/cloud_cost_management/realtime_costs.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Realtime Costs
```

Comment:
```suggestion
title: Real-Time Costs
```

Reply:
```suggestion
title: Real-Time Cloud Costs
```

I think putting "cloud" in the title is actually more clear. There are a lot of costs that we would not be covering